add-on/panels/js/roomStore.js

```diff
@@ -352,12 +355,25 @@ loop.store = loop.store || {};
      */
     facebookShareRoomUrl: function(actionData) {
       var encodedRoom = encodeURIComponent(actionData.roomUrl);
-      loop.request("GetLoopPref", "facebook.shareUrl")
-        .then(shareUrl => {
-          loop.request("OpenURL", shareUrl.replace("%ROOM_URL%", encodedRoom));
-        }).then(() => {
-          loop.request("NotifyUITour", "Loop:RoomURLShared");
-        });
+
+      loop.requestMulti(
+        ["GetLoopPref", "facebook.appId"],
+        ["GetLoopPref", "facebook.fallbackUrl"],
+        ["GetLoopPref", "facebook.shareUrl"]
+      ).then(results => {
+        var app_id = results[0];
+        var fallback_url = results[1];
+        var redirect_url = encodeURIComponent(actionData.originUrl ||
```

Comment:
What's `originUrl`? `FacebookShareRoomUrl` action seems to only have `from` and `roomUrl`